charts/ndb-operator/values.yaml

```diff
@@ -17,7 +17,7 @@ serviceAccount:
   annotations: {}
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
-  name: "ndb-operator-service-account"
+  name: "ndb-operator-controller-manager"
```

Comment:
Needed to update this based on the error message seen during the installation of the operator using helm. I am further checking why it is looking for a specific name to work.

Reply:
Discussed this with Christophe. He informed that this name is picked up from "kind: ServiceAccount" file, where the name specified was "ndb-operator-controller-manager". After updating it to "ndb-operator-service-account" in that file, it worked. It was also decided that since service account name is internal to ndb-operator, we should remove the it from templating logic.